Library/Formula/postgresql.rb

```diff
@@ -1,9 +1,11 @@
 class Postgresql < Formula
   homepage "http://www.postgresql.org/"
+  revision 1
 
   stable do
     url "http://ftp.postgresql.org/pub/source/v9.4.0/postgresql-9.4.0.tar.bz2"
     sha256 "7a35c3cb77532f7b15702e474d7ef02f0f419527ee80a4ca6036fffb551625a5"
+    version "9.4"
```

Comment:
It's choking at `plist_options :manual => "postgres -D #{HOMEBREW_PREFIX}/var/postgres-#{version_suffix}"` so I think this is related to Ruby class definition vs. instantiation and class vs. instance methods; adding a second `self.version_suffix` method keeps it from throwing exceptions. I still don't know Ruby classes well enough to tell if that's a good solution. :)